AmplifyPlugins/DataStore/AWSDataStoreCategoryPluginTests/Core/SQLiteStorageEngineAdapterTests.swift

```diff
@@ -375,4 +375,120 @@ class SQLiteStorageEngineAdapterTests: BaseDataStoreTests {
             XCTFail(String(describing: error))
         }
     }
+
+    func testClearIfNewVersionShouldStoreInitialVersion() {
```

Comment:
- no longer storing initial version, the scenario is "empty user defaults" like `testClearIfNewVersionWithEmptyUserDefaults`